src/test/java/org/opensearch/neuralsearch/query/aggregation/MetricAggregationsWithHybridQueryIT.java

```diff
@@ -81,103 +81,103 @@ public class MetricAggregationsWithHybridQueryIT extends BaseAggregationsWithHyb
      */
     @SneakyThrows
     public void testWithConcurrentSegmentSearch_whenAvgAggs_thenSuccessful() {
-        updateClusterSettings(CLUSTER_SETTING_CONCURRENT_SEGMENT_SEARCH, true);
```

Comment:
Do not review this class as it already merged in main. The change is showing here because we have cherry picked the commit.